assets/src/design-system/components/dropDown/select/components.js

```diff
@@ -46,6 +46,13 @@ export const SelectButton = styled.button(
     text-overflow: ellipsis;
     cursor: pointer;
 
+    :focus {
```

Comment:
Let's add a comment here just so we know why this was added 👍

Reply:
Good idea, I've referenced this PR as well. I'd like to know in case there is a better solution to this. It didn't seem to affect the way the outline is applied in the Safari, Firefox, and Chrome, so it seems fine for the requirement, but maybe there is a better way to do this or indicates something we're doing wrong.

I did notice a regression, clicking on the select now activates the focus state where it didn't before. I'm going to fix that as well. Hopefully can be done with some clever css.